src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -370,20 +370,20 @@ def load_standard_data(key: str, location: Union[str, List[int]]) -> pd.DataFram
     ]
 
     if key in use_2019_data_keys:
-        data = interface.get_measure(entity, key.measure, location, True)
+        data = interface.get_measure(entity, key.measure, location, 2019)
```

Comment:
I might suggest replacing this with a constant but it looks like it rarely appears?

Reply:
Typically I'd agree, though they are in very specific 2019-related if blocks so I care less